terraform/module-prometheus/prometheus.tf

```diff
@@ -71,45 +71,42 @@ resource "aws_security_group_rule" "prometheus_to_himself_instance_metrics" {
 ###
 
 resource "aws_instance" "prometheus" {
-  ami = "${data.aws_ami.debian.id}"
+  ami = data.aws_ami.debian.id
 
   # associate_public_ip_address = false
   count                = 1
```

Comment:
I dont remember, why do we have `count = 1` without variable ?
Should we remove the count ? Should we add a variable ?

Reply:
`var.count` was used in name, tags etc, so declaring `count = 1` is mandatory in 0.12.